stacks/HANA-Scaleout-Standby/awx/HANA-Scaleout/extra-vars.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
The awx subdirectory should be HANA-Scaleout-Standby, otherwise the AWX job HANA-Scaleout is going to be overwritten.

Reply:
Done. Updated the name.